src/index.html

```diff
@@ -17,11 +21,401 @@
     />
   </head>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="header__logo-img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
-  <body>
-    <h1>Catalog</h1>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__product-image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        />
+
+        <h2 class="card__product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__product-rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="reviews-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__product-price">
+          <p class="price-title">Price:</p>
+          <p class="price-numbers">$2,199</p>
+        </div>
+
+        <a
+          class="card__button-buy"
+          data-qa="card-hover"
```

Comment:
According to the checklist, the BEM modifier should be `--active` instead of `is-active`. Please update the class name to follow the BEM naming convention.